README.md

```diff
@@ -184,68 +204,82 @@ parameter a `Term`. Their ASTs are defined
 ### Printing
 
 The [`printer`](/src/main/scala/smtlib/printer) helps with printing out SMT-LIB
-complient commands. This means that the output of a printer can be send
+compliant commands. This means that the output of a printer can be sent
 directly to an SMT solver.
 
 ### Standard Theories
 
-Finally the [`theories`](/src/main/scala/smtlib/theories) module provides tree
-builders to create theory-specific formulas. Each theory module provides
-`apply` and `unapply` methods on various object to manipulate the `Term`
-representing the actual theory expression.
+Finally, the [`theories`](/src/main/scala/smtlib/theories) module provides tree
+builders to create theory-specific formulas. Each theory module provides `apply`
+and `unapply` methods on various object to manipulate the `Term` representing
+the actual theory expression.
 
 Development
 -----------
 
-The library is still under development and the API will likely go through quite a few
-changes. It was originally part of [CafeSat](https://github.com/regb/scabolic)
-and has been made standalone in order for the
-[Leon](https://github.com/epfl-lara/leon) project to rely on it.
+The library is still under development and the API will likely go through quite
+a few changes. It was originally part of
+[CafeSat](https://github.com/regb/cafesat) and has been made standalone in order
+for the [Inox](https://github.com/epfl-lara/inox) and
+[Stainless](https://github.com/epfl-lara/stainless) projects to rely on it.
 Hopefully, it can be useful to other people as well.
 
 ### Testing
 
-In order to attain a decent level of quality, there is a relatively strict policy for testing.
-Testing is separated in two levels of testing: unit tests and integration tests. Unit test
-are dependency free and run entirely in memory. While integration tests will rely on things
-like file system and external solvers. Unit test are very fast and should be easy to run as part
-of a regular build cycle (you can run them after each compile), while integration tests are a bit
-slower and are meant to pass on each commit.
+In order to attain a decent level of quality, there is a relatively strict
+policy for testing. Testing is separated in two levels of testing: unit tests
+and integration tests. Unit test are dependency free and run entirely in memory.
+While integration tests will rely on things like file system and external
+solvers. Unit test are very fast and should be easy to run as part of a regular
+build cycle (you can run them after each compile), while integration tests are a
+bit slower and are meant to pass on each commit.
 
 In SBT, the command `sbt test` will run the unit test suite, while the command
-`sbt it:test` will run the integration test suite. During developement, it
-should be fine to run in mode `~testQuick`.
+`sbt IntegrationTest/test` will run the integration test suite. During
+development, it should be fine to run in `~testQuick` mode.
```

Comment:
Maybe add an example command for the testQuick mode

Reply:
```suggestion
development, it should be fine to run in [quick-testing mode](https://www.scala-sbt.org/1.x/docs/Testing.html#testQuick), as `sbt ~testQuick`, running only previously failing tests or those expected to change.
```

ok?